src/common/utils/time/time-utils.ts

```diff
@@ -1,5 +1,17 @@
 import moment from 'moment';
 
+export const timeFormats = {
+  YYYY_MM_DD: 'YYYY-MM-DD',
+  MMM_DO_YYYY: 'MMM Do, YYYY',
+  MMM_DD_YYYY: 'MMM DD, YYYY',
+  LL: 'LL',
+  L: 'L',
+};
```

Comment:
Consider adding an example of each format besides them, also let's make it an enum instead:

```ts
export enum TimeFormats = {
  YYYY_MM_DD = 'YYYY-MM-DD',     // 2021-03-17
  MMM_DO_YYYY = 'MMM Do, YYYY',  // Mar 17th, 2021
  // ...
};
```